.gitignore

```diff
@@ -12,3 +12,5 @@ develop-eggs
 .installed.cfg
 lib
 lib64
+
```

Comment:
Can we add comments in `.gitignore`? If yes, then please add relevant comments so that the distinction between 2 sections are made in future too